### PR TITLE
Notes

### PR DESCRIPTION
**Pull request by mohammed-sas**
### Feature
1.user can add notes to videos in single video page
2.user can edit notes
3.user can delete notes
4.on add/edit of notes , the video would be paused
5.user can see the timestamp on notes

### CODE REVIEW

**Review by Abhijeet43 — APPROVED** [review]
- The functionality is working as intended!!!
- Good Naming Conventions
- Directory Structure is clean

File: src/pages/single video/notes/modal/EditNotesModal.jsx [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+        onSubmit={submitHandler}
+      >
+        <i
+          onClick={setShowEditForm}
+          className={`fas fa-times text-white ${classes["close-btn"]}`}
+        ></i>
+        <h3 className="text-white">Edit note</h3>
+        <label htmlFor="title">
+          Title <br />
+          <input
```
Comment (kart-c) [inline src/pages/single video/notes/modal/EditNotesModal.jsx:42]: you can make either the label or input a block element and avoid the `<br />` tag

**Review by kart-c — APPROVED** [review]
- Functionality is working as expected
- Really like that you are showing the user the timestamp at which the note was written 
- Also like how all your note handlers(addnote, deletenote etc) are in one place, in notes-context itself

File: src/pages/single video/notes/modal/EditNotesModal.jsx (hunk 1 above)
Comment (mohammed-sas) [reply src/pages/single video/notes/modal/EditNotesModal.jsx:42]: sure i'll avoid br tags in final refactor

**Pull request merged**